dataprofiler/profilers/profile_builder.py

```diff
@@ -683,3 +687,127 @@ def tqdm(l):
             
         return profile
 
+    def _delete_data_labelers(self):
```

Comment:
Is there a better way to delete and restore all data labelers that have been created?